src/app/modules/ewt-patron/enrolment/enrolment-status-check/enrolment-status-check.component.html

```diff
@@ -1,4 +1,11 @@
 <ng-container [ngSwitch]="getStatus$ | async">
+  <p *ngIf="!(isSynced$ | async)">Your current wallet
+    <ng-container *ngIf="getAddress$ | async as address">
+      <span [matTooltip]="address">{{ address | didFormatMinifier}}</span>
+    </ng-container>
+    is not authorized to stake. If this is
```

Comment:
Should't this part be added inside the <ng-container as well ? so it renders in the same time with getAddress$?

Reply:
It might be added, but place that is toggling this depends on role enrolment status. Without this address we can still see message.

Reply:
then it's ok